src/stories/molecules/Dropdown/Dropdown.style.ts

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const DropCaretWrapper = styled.span`
+    margin: 0 0 0 1rem;
```

Comment:
Just a question for you to ponder: _What's the difference between using `rem` and `em`?_